patterns/1-initial/extensions-to-manage-contributions-at-scale.md

```diff
@@ -0,0 +1,81 @@
+## Title
+
+Extensions to Manage Contributions at Scale
+
+## Patlet
+
+TBD
+
+## Problem
+
+As the number of new feature contributions to a mature InnerSource repository rapidly increases, it adds more burden on code reviews and maintenance. How do we allow for faster release of new features, encouraging innovation and experimentation; while also keeping repositories well maintained?
+
+## Context
+
+- More employees are contributing to strategic InnerSource codebases, some of which are scaling rapidly with new feature contributions. This is adding a huge code review backlog on a smaller group of maintainers, resulting in maintainer burnout as well as slowing down release of new capabilities.
```

Comment:
```suggestion
- More employees are contributing to a strategic InnerSource codebase, which is scaling rapidly with new feature contributions. This is adding a huge code review backlog on a smaller group of maintainers, resulting in maintainer burnout as well as slowing down release of new capabilities.
```

I would use singular instead of plural here
i.e. this patter explains is by the example one core repository/project/codebase.

Of course there can be multiple projects within an org that are in the same situation. However then the pattern is just applied multiple times, right?

If you accept this suggestion, we have to review the following bullets for use of singular/plural as well.

Reply:
yes that is correct, thanks. I will take a pass and edit all